FinSearchAPI/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -335,8 +335,13 @@ protected function setImages()
                 }
 
                 if (count($imageDetails) > 0) {
-                    $imagePath = $mediaService->getUrl($imageDefault);
-                    $imagePathThumb = $mediaService->getUrl(array_values($imageDetails)[0]);
+                    $replacements = array(
```

Comment:
- [ ] Use `[]` notation instead of `array`. We can safely assume that the first one will be supported by the client's PHP
- [ ] Move this array to the beginning of this method. Since it stays the same for every image, there is no need to initialize it in every iteration.

Reply:
@ihanli Done! :+1: